src/main/java/Lists.java

```diff
@@ -1,38 +1,38 @@
 import java.util.ArrayList;
 
 public class Lists {
-  private static ArrayList<Task> list = new ArrayList<Task>();
+    private static ArrayList<Task> list = new ArrayList<Task>();
```

Comment:
consider using naming `tasks` instead of `list` to better represent this variable

```suggestion
    private static ArrayList<Task> tasks = new ArrayList<Task>();
```